supportedBackends/google/pipelines/v2alpha1/src/main/scala/cromwell/backend/google/pipelines/v2alpha1/api/EgressCheckAction.scala

```diff
@@ -0,0 +1,33 @@
+package cromwell.backend.google.pipelines.v2alpha1.api
+
+import com.google.api.services.genomics.v2alpha1.model.{Action, Mount}
+import cromwell.backend.google.pipelines.common.action.ActionLabels._
+import cromwell.backend.google.pipelines.common.PipelinesApiConfigurationAttributes.GcsTransferConfiguration
+import cromwell.backend.google.pipelines.common.api.PipelinesApiRequestFactory.CreatePipelineParameters
+import cromwell.backend.google.pipelines.common.PipelinesApiJobPaths._
+import cromwell.backend.google.pipelines.v2alpha1.api.ActionBuilder.cloudSdkShellAction
+import cromwell.backend.google.pipelines.v2alpha1.api.ActionCommands.localizeFile
+
+trait EgressCheckAction {
+  def egressCheckAction(createPipelineParameters: CreatePipelineParameters, mounts: List[Mount])
+                       (implicit gcsTransferConfiguration: GcsTransferConfiguration): List[Action] = {
```

Comment:
This should be optimized so that if regionality is global and strict is false (as would be the case for all existing Terra submissions) then no regionality-checking actions would be created or executed.